docs/pages/admin-guides/management/operations/breakglass-access.mdx

```diff
@@ -0,0 +1,197 @@
+
+1. The Teleport Agent on a server has crashed, gone offline, or become unusable.
+2. The Teleport control plane is down and cannot be used to access systems, and out-of-band access is necessary to fix it.
+
+## How it works
+
+Teleport's CA can issue short-lived, signed certificates that can be used to grant access to OpenSSH servers in emergency disaster recovery scenarios. By configuring the OpenSSH server on the Teleport Agent to trust Teleport’s CA, users with valid Teleport-issued certificates can authenticate to the server without requiring static SSH keys or passwords even if Teleport itself is down.
+
+Below, we’ll detail the steps to accomplish the following objectives:
+
+1. Configure the Teleport Agent's OpenSSH server to trust Teleport's CA
+2. Create a `breakglass` system user on the remote host
+3. Restrict cert-based SSH access to this `breakglass` user only
+4. Create a `breakglass` Role and User in Teleport
+5. Generate `breakglass` SSH Key and Cert using Teleport's CA
+6. Access the remote server using a Teleport issued cert even if Teleport is down or unresponsive
+
+## Prerequisites
+
+- OpenSSH client version 7.4 or above on your local machine.
+- A Linux host with OpenSSH server (`sshd`) version 7.4 or above.
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- A Teleport user with ability to add new `impersonate` role permissions.
+
+## Step 1/6. Configure sshd to trust the Teleport CA
+
+For break glass access, the OpenSSH server must be configured to trust client certificates issued by the Teleport Certificate Authority (CA).
+
+1. **Export the Teleport CA public key:**
+
+On the OpenSSH server, run the following command. Replace `proxy.example.com` with the address of your Teleport Proxy Service:
+
+```bash
+export KEY=$(curl 'https://proxy.example.com/webapi/auth/export?type=openssh' | sed "s/cert-authority\ //")
+```
+
+2. Make the public key accessible to sshd:
+
+```bash
+sudo bash -c "echo \"$KEY\" > /etc/ssh/teleport_openssh_ca.pub"
+sudo bash -c "echo 'TrustedUserCAKeys /etc/ssh/teleport_openssh_ca.pub' >> /etc/ssh/sshd_config"
+```
+
+3. Restart the sshd service:
+
+```bash
+sudo systemctl restart sshd
+```
+
+Now, `sshd` will trust users who present a Teleport-issued SSH certificate.
+
+<Admonition type="important">
+Complete Steps 4 and 5 below to ensure only the `breakglass` user is allowed to log in with a valid Teleport certificate and is configured with proper access restrictions, in order to prevent user switching once logged in. 
+</Admonition>
+
+## Step 2/6. Create a `breakglass` user on the Teleport Agent server
+
+Log into the Teleport Agent server and run the following commands.
+
+1. Create the `breakglass` user:
+
+```bash
+sudo adduser breakglass
+```
+
+Set a password for `breakglass` when prompted. We recommend the use of a long, random string as this password will never be used for access.
```

Comment:
If we're not going to allow passwordful access to the user and the password is throwaway there's no point to having a password, we should use `adduser --disabled-password` or something like that.

If the password will be needed for privilege escalation through `sudo` we might as well save the password in the same vault where we store the access cert, and potentially require the password for login in addition to the cert.

Reply:
> adduser --disabled-password

Oooh cool, didn't know this was an option. I'll add in the feedback